docs/examples/DataSet/Dataset Context Manager.ipynb

```diff
@@ -7,7 +7,9 @@
     "# The Context Manager aka the Measurement Object\n",
     "\n",
     "This notebook shows some ways of performing different measurements using \n",
-    "QCoDeS parameters and the new DataSet accessed via a context manager.\n"
+    "QCoDeS parameters and the new DataSet accessed via a context manager. Here, it is assumed that the reader has some degree of familiarity with fundamental objects and methods of QCoDeS.\n",
```

Comment:
When adding some requirements to the reader here, I would be more specific, just along the lines as Harshit is saying: name the concepts required to understand and link to the documents where they can be learned.

Reply:
Well, I would prefer not doing that as every required concept will surely have some additional dependencies and prerequisities, as well. Having a general statement or none (as in the previous version) is sufficient, IMHO. Harshit's above point is OK because the concept is explicitly referred and thus should be linked :)

Reply:
I am fine with this, you have the better overview, Umit.